packages/p2p/src/components/advertiser-page/advertiser-page.scss

```diff
@@ -0,0 +1,99 @@
+.advertiser-page {
+    &__container {
+        display: flex;
+        flex-flow: column;
+        height: calc(100vh - 47px - 53px - 60px - 52px - 36px - 2.4rem - 1.6rem);
+        min-height: 555px;
+        overflow-y: scroll;
+    }
+
+    &__header {
+        display: inherit;
+        height: fit-content;
+        justify-content: space-between;
+        width: 657px;
+
+        &-avatar {
+            align-items: center;
+            border-radius: 50%;
+            color: var(--text-colored-background);
+            display: flex;
+            font-size: 12px;
```

Comment:
for font-size, margin, and padding, we can use `rem` instead imo

Reply:
Changed to `rem` for font-size, margin and padding 👍